WIP #601: removed dml user mechanism (#603)
factcast/factcast: factcast-site/documentation/content/migration/_index.md

There is a [section on optimitic locking](/usage/java/optimistic_locking/) as a new api feature.

#### Postgres module uuid-ossp 

> The Postgres module *uuid-ossp* is necessary for the new optimistic locking api feature. In order to provide a user

Reviewer: ```suggestion
The Postgres module *uuid-ossp* is necessary for the new optimistic locking api feature. In order to install this
```

> who is a superuser. 

Reviewer: ```suggestion
extension, the user performing the Liquibase operations requires Postgres superuser permissions. 
```
